tool_collections/cufflinks/cuffmerge/.shed.yml

```diff
@@ -1,4 +1,8 @@
-# repository published to https://toolshed.g2.bx.psu.edu/repos/devteam/cuffmerge
-owner: devteam
+categories:
+- Transcriptomics
+description: Galaxy wrappers for the Cuffmerge tool.
+homepage_url: http://cole-trapnell-lab.github.io/cufflinks/
 name: cuffmerge
```

Comment:
long_description: |
  Cuffmerge merges together several Cufflinks assemblies and automatically filters a number of transfrags that are   
  probably artfifacts. If you have a reference GTF file available, you can provide it to the script in order to gracefully merge 
  novel isoforms and known isoforms and maximize overall assembly quality.